src/ui-kit/components/Button/index.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button as SemanticButton } from 'semantic-ui-react';
+import styled from 'styled-components';
+import { brandPrimaryColor, secondaryColor, white } from '../../theme';
```

Comment:
please separate import groups with space. The last import should be a separate import group.